### PR TITLE
fix Issue 18372 - REG(2.072): error missing line number: Error: undef…

### PR DESCRIPTION
**Pull request by WalterBright**
…ined identifier __va_list_tag

It's only sort of a regression.

### CODE REVIEW

File: test/fail_compilation/fail18372.d [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*
+DISABLED: win32 win64 linux32 osx freebsd
```
Comment (thewilsonator) [inline test/fail_compilation/fail18372.d]: You've disabled this on every platform.

Comment (thewilsonator) [reply test/fail_compilation/fail18372.d]: Oh, this runs on linux64 only?

File: test/fail_compilation/fail18372.d [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+TEST_OUTPUT:
+---
+Error: `__va_list_tag` is not defined, perhaps `import core.stdc.stdarg;` ?
```
Comment (jacob-carlborg) [inline test/fail_compilation/fail18372.d]: How is this fixing the issue? The issue is about missing line number in the error message.

File: test/fail_compilation/fail18372.d (hunk 1 above)
Comment (jacob-carlborg) [reply test/fail_compilation/fail18372.d]: The error occurs on macOS as well.

File: test/fail_compilation/fail18372.d (hunk 2 above)
Comment (WalterBright) [reply test/fail_compilation/fail18372.d]: Indeed I was so focused on the message I overlooked the line number.

File: test/fail_compilation/fail18372.d (hunk 1 above)
Comment (WalterBright) [reply test/fail_compilation/fail18372.d]: It isn't really necessary to run it on all the posix platforms.

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**